src/external/json.hpp

```diff
@@ -30,6 +30,11 @@ SOFTWARE.
 #ifndef INCLUDE_NLOHMANN_JSON_HPP_
 #define INCLUDE_NLOHMANN_JSON_HPP_
 
+#if defined(__GNUC__) && !defined(__clang__)
+#pragma GCC diagnostic push
+#pragma GCC diagnostic ignored "-Wmaybe-uninitialized"
+#endif
```

Comment:
why edit the json header ?

Reply:
Because gcc emits warning when compiling it since it is not longer considered a system header after https://github.com/cosmo-epfl/librascal/pull/165/commits/32ea71755bcf39f8648d7a68950177bafd84638e